Rakefile

```diff
@@ -0,0 +1,10 @@
+require 'minitest/autorun'
+load_paths = FileList['spec', 'lib']
+ruby_args = load_paths.pathmap('-I%p')
+
+task default: %w[test]
+
+task :test do
+  command = "ruby #{ruby_args} -rminitest/autorun spec/core_spec.rb"
+  exec command
+end
```

Comment:
rake will run the default, which happens to be the test task.  The newer pull request adds in a style task, which will run rubocop for you.  That is accomplished by running `rake style` in the terminal.

You can and should research rake to create the tasks that you would like to have, perhaps to auto-correct the style offenses or whatever it is you would like to have.